manifests/02-role.yaml

```diff
@@ -8,9 +8,14 @@ rules:
 - apiGroups:
```

Comment:
we don't want to be using this file -- only update the manifest itself

Reply:
I suggest to keep them both in sync until someone picks to verify & switch the usage to CSV manifest only and finally delete them. IMHO, for the one doing this job checking differences makes the job more difficult in the end.

Reply:
@bparees did a demo on just this situation with the new olm-based testing. These files are not being used anymore and we have tech-debt to remove them.

Reply:
Alright! Blessed the one to remove tech debt then.